### PR TITLE
RavenDB-13311 Cluster observer should be more informative for the rea…

### PR DESCRIPTION
**Pull request by karmeli87**
…son of why not promoting a node to become a member

### CODE REVIEW

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs [hunk 1]
```diff
@@ -137,28 +137,28 @@ public void Run(CancellationToken token)
...
-            if (_logger.IsInfoEnabled || _logger.IsOperationsEnabled)
+            if (_lastLogs.TryGetValue(message, out var last))
             {
-                if (_lastLogs.TryGetValue(message, out var last))
-                {
-                    if (last + 10 > _iteration)
-                        return;
-                }
-                _lastLogs[message] = _iteration;
+                if (last + 60 > _iteration) // 30 sec
```
Comment (ayende) [inline src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs]: I don't follow this computation?

Comment (karmeli87) [reply src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs]: each iteration happened every 500 ms.
500ms * 60 = 30 s

Comment (ayende) [reply src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs]: Can you add this as a comment?

Comment (karmeli87) [reply src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs]: done

**Review by ayende — APPROVED** [review]

**Pull request merged**